lib/routes/admin.js

```diff
@@ -52,6 +52,9 @@ router.route('/deployments/:name/backup')
 router.route('/deployments/:name/backup/status')
   .get(controller.handler('getLastOobBackupStatus'))
   .all(middleware.methodNotAllowed(['GET']));
+router.route('/deployments/:name/config')
```

Comment:
@muralisc / @jagadish-kb  : should we have bosh in the url path or keep it generic as it is now ? Because tomorrow we might have some other configs as well , correct ?

Reply:
yeah makes sense . Now its not clear which config it is.
```
/deployments/<name>/directorconfig
```

Reply:
how about this : `/deployments/<name>/director` obviously from the URL you cant get anything other than config, so no need to explicitly specify it.